.goreleaser.yml

```diff
@@ -0,0 +1,32 @@
+project_name: chronosphere
+before:
+  hooks:
+  - go mod tidy
```

Comment:
is this needed? Ideally we'd validate in CI that `go mod tidy` doesn't result in any changes to go.mod/go.sum so every change on master has already run this

only concern is whether it can lead to a dirty repo if it makes a change and how that affects the built version/tag.

Reply:
Goreleaser fails if the repo is dirty, so this prehook is one last check to ensure all of our deps are correct. This occurs _inside_ the container as opposed to outside, so if for whatever reason there is a difference this will catch it.